rest-service/manager_rest/rest/resources_v3_1/filters.py

```diff
@@ -158,6 +165,12 @@ def _get_filter_rules_by_type(filter_rules_list, filter_rule_type):
             filter_rules_list if filter_rule['type'] == filter_rule_type]
 
 
+def _verify_not_a_system_filter(filter_elem):
+    if filter_elem.is_system_filter:
+        raise manager_exceptions.IllegalActionError(
+            'Cannot delete a system filter')
```

Comment:
probably "update" instead of "delete" when called from patch?

Reply:
Oh right, thanks!